README.rst

```diff
@@ -55,6 +55,46 @@ You can also get random types of strings by excluding the second argument:
         assert len(value) > 0
 
 
+Additionally, you can run tests with different string lengths by passing in a list of lengths:
+
+.. code-block:: python
+
+    @pytest.mark.faux_string(4, 'alpha', length=[5, 15])
+    def test_gen_alpha_string_with_variable_length(value):
+        """Generate an `alpha` string of length of either 5 or 15."""
+        assert len(value) == 5 or len(value) == 15
+
+This will generate 4 new tests
+
+::
+
+    tests/test_faux_string.py::test_gen_alpha_string_with_variable_length[EAppa] PASSED                                                                                                                                          [ 91%]
```

Comment:
I can, though I'm not sure what use having these example tests is now that the useful part (generated strings) is no longer presented. It might be better just to drop this part from the readme altogether.

Reply:
The usefulness imo is that people can see how to use it (and what the output would be).

Reply:
Oh I didn't mean removing the actual tests themselves, just the pytest run output. In the mean time I can update them to the format you requested, and we can discuss their inclusion another day!